### DISCUSSION TITLE
Git build test failure

### DISCUSSION THREAD

**Discussion by inertia666** [DevSkillsHQ/discusssions#26] [Help] [answered]
I have completed my project and the server runs locally. The backend npm test thing works and all my tests pass locally.

The test is not working with the git pipeline. I don't think it's finding the server or it's not running in the container.

info: Microsoft.Hosting.Lifetime[14]
      Now listening on: http://localhost:5000
info: Microsoft.Hosting.Lifetime[0]
      Application started. Press Ctrl+C to shut down.
info: Microsoft.Hosting.Lifetime[0]
      Hosting environment: Development
info: Microsoft.Hosting.Lifetime[0]
      Content root path: /home/runner/work/lookup-service-level-2_f0e4b00-agf259/lookup-service-level-2_f0e4b00-agf259/app/backend
http://localhost:56790/ping timed out on retry 91 of 3, elapsed 90306ms, limit 90000ms
Error: connect ECONNREFUSED 127.0.0.1:56790

I haven't changed anything from the boilerplate code. The server should work on both http and https

![image](https://github.com/DevSkillsHQ/discusssions/assets/59329721/248b6f14-9bc7-46d6-8c89-09da966a73b4)

**Comment by fenske** [comment, marked as answer]
Closing this discussion as we covered this in https://github.com/orgs/DevSkillsHQ/discussions/25.
